docs/config.md

```diff
@@ -0,0 +1,132 @@
+# `holograph config`
+
+Initialize the Holograph CLI with a config file. If no flags are passed, the CLI will prompt you for the required information.
+
+- [`holograph config`](#holograph-config)
+- [`holograph config:networks`](#holograph-confignetworks)
+- [`holograph config:user`](#holograph-configuser)
+- [`holograph config:view`](#holograph-configview)
+
+## `holograph config`
+
+Initialize the Holograph CLI with a config file. If no flags are passed, the CLI will prompt you for the required information.
+
+```
+USAGE
+  $ holograph config [--url <value> --network
```

Comment:
having all these networks listed seems kind of verbose and hard to maintain especially if the list keeps growing over time